editor.planx.uk/src/pages/Team.tsx

```diff
@@ -287,6 +301,21 @@ const FlowItem: React.FC<FlowItemProps> = ({ flow, teamId, refreshFlows }) => {
                 handleCopy();
               },
             },
+            {
+              label: "Move",
+              onClick: () => {
+                const newTeam = prompt("New team");
```

Comment:
ideally, this would probably be a clever dropdown of all existing team slugs besides the current one to select from - but a `prompt` is a simpler MVP implementation here and I think we can safely assume that only experienced/familiar content editors are going to be "moving" flows right now anyways

Reply:
Yeah I think this is totally fine for now whilst we have a small number of teams, and a small number of editors using this 👍